alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/accessor/SettingsUtility.java

```diff
@@ -9,23 +9,12 @@
 
 import java.util.Optional;
 
-import com.synopsys.integration.alert.api.common.model.exception.AlertException;
-import com.synopsys.integration.alert.common.persistence.model.ConfigurationModel;
-import com.synopsys.integration.alert.common.rest.model.FieldModel;
+import com.synopsys.integration.alert.common.rest.model.SettingsProxyModel;
 import com.synopsys.integration.alert.descriptor.api.model.DescriptorKey;
 
 public interface SettingsUtility {
-
+    //TODO: This is used in AlertStartupInitializer::initializeConfigs and is set to be Deprecated
```

Comment:
Let's just use the DescriptorMap that's in AlerrtStartupInitializer to lookup the key where the getKey method of this interface is used.

`descriptorMap.getDescriptorKey(SettingsDescriptoKey.SETTINGS_COMPONENT);`

Then we can completely delete this interface.

Reply:
We need this interface because it's in Alert common and it's used as a marker interface correct?

Reply:
SettingsDescriptorKey is unavailible in the AlertStartupInitializer so it is unfortunately still required here. Likewise, since we need to implement the getConfiguration method we cannot make this a marker interface.